lazy_tensor_core/lazy_bench.py

```diff
@@ -675,17 +676,21 @@ def save_error(name, test, error, dir):
         launch_command = f"python {' '.join(copy_argv)} --run_in_subprocess '{model_name}' --output_dir={dirpath}"
         env = os.environ
         env["LTC_TS_CUDA"] = "1"
-        try:
-            rc = subprocess.run(launch_command,
+        proc = subprocess.Popen(launch_command,
                         env=env,
-                        timeout = args.timeout,
                         shell=True,
                         stderr=subprocess.STDOUT)
+        try:
+            outs, errs = proc.communicate(timeout=args.timeout)
         except subprocess.TimeoutExpired:
```

Comment:
@Krovatkin, per our discussion in workchat, we probably want to do the clean up for all exceptions.

Reply:
I think the timeout exception is special because it's triggered in master rather than in a child process. I'd hope if the child process catches an exception and exits normally, we should be releasing the memory we allocated in the child process.  @wconstab  wdyt?

Reply:
no idea. i guess it doesn't hurt to clean up conservatively.  has this version of the PR been shown to fix the issue so far in CI torchbench?